x/auth/tx/builder.go

```diff
@@ -157,10 +157,6 @@ func (w *wrapper) GetMemo() string {
 	return w.tx.Body.Memo
 }
 
-func (w *wrapper) GetSignatures() [][]byte {
-	return w.tx.Signatures
-}
```

Comment:
This function is not used, it's probably an artifact from an old code. The `wrapper` is a private structure exposed as `TxBuilder` interface, which doesn't define this method at all.
So, either we add this function to the interface, or remove it to avoid that someone will like to hack with an unsupported code. I decided to remove it and hopefully later reconsider refactoring the `TxBuilder` to make it more user friendly.